solutions/instances/outputs.tf

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "log_analysis_name" {
+  value       = module.observability_instance.log_analysis_name
+  description = "The name of the provisioned Log Analysis instance."
+}
+
+output "log_analysis_crn" {
+  value       = module.observability_instance.log_analysis_crn
+  description = "The id of the provisioned Log Analysis instance."
+}
+
+output "log_analysis_guid" {
+  value       = module.observability_instance.log_analysis_guid
+  description = "vaThe guid of the provisioned Log Analysis instance."
+}
+
+output "log_analysis_ingestion_key" {
+  value       = module.observability_instance.log_analysis_ingestion_key
+  description = "Log Analysis ingest key for agents to use"
+  sensitive   = true
+}
+
+output "cloud_monitoring_name" {
+  value       = module.observability_instance.cloud_monitoring_name
+  description = "The name of the provisioned IBM cloud monitoring instance."
+}
+
+output "cloud_monitoring_crn" {
+  value       = module.observability_instance.cloud_monitoring_crn
+  description = "The id of the provisioned IBM cloud monitoring instance."
+}
+
+output "cloud_monitoring_guid" {
+  value       = module.observability_instance.cloud_monitoring_guid
+  description = "The guid of the provisioned IBM cloud monitoring instance."
+}
+
+output "cloud_monitoring_access_key" {
+  value       = module.observability_instance.cloud_monitoring_access_key
+  description = "IBM cloud monitoring access key for agents to use"
+  sensitive   = true
+}
+
+output "cos_bucket_details" {
+  value       = (var.existing_log_archive_cos_bucket_name == null || var.existing_at_cos_target_bucket_name == null) ? module.cos[0].buckets : null
+  description = "The details of the COS buckets."
+}
```

Comment:
@iamar7 This is only going to output details from the first bucket right? I think we should separate outputs for each of the buckets being created

Reply:
@ocofaigh it contains the output of both the buckets so do we want to separate the outputs of each of the bucket?

Reply:
@iamar7 yes for Stacks to work, outputs need to be strings

Reply:
We don't need to output all bucket details - infact we might be fine to just output the names for now